person.rb

```diff
@@ -0,0 +1,22 @@
+# Class Person file.
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def can_use_services?
+    true if Person._is_of_age? || Person.parent_permission
```

Comment:
- [ ] I get a **NoMethodError** error referring to `Person._is_of_age?` when I try to call `can_use_services?`. This is because what you defined under private is `of_age` but you are calling `_is_of_age` which does not exist. I also get the same error for `Person.parent_permission` because parent_permission is not defined for the Person class. I would suggest removing the `Person.` so your line looks something similar to this:
  ```
    true if of_age? || @parent_permission
  ```

  ![rami-nomethod](https://user-images.githubusercontent.com/36542986/145880244-ae9ebeb0-f966-4bd5-97ab-2410eb4cf797.jpg)